examples/SimpleAudioPlayer/SimpleAudioPlayer.ino

```diff
@@ -38,14 +38,15 @@ void setup() {
 
   // 44100kHz stereo => 88200 sample rate
   // 100 mSec of prebuffering.
-  Audio.begin(88200, 100);
+  //Audio.begin(88200, 100);
```

Comment:
```suggestion
```
The commented out code just clutters the sketch. We can always look at the commit history if we need to see how the code used to be.